src/Database/LSMTree/Internal/Snapshot.hs

```diff
@@ -61,7 +61,7 @@ import           System.FS.BlockIO.API (HasBlockIO)
 newtype SnapshotLabel = SnapshotLabel Text
   deriving stock (Show, Eq)
 
-data SnapshotTableType = SnapNormalTable | SnapMonoidalTable
+data SnapshotTableType = SnapNormalTable | SnapMonoidalTable | SnapFullTable
```

Comment:
![image](https://github.com/user-attachments/assets/869ebddb-1ee0-4c30-8f6f-268742a2b4e7)

Long term, we should probably either have one or two types of tables, but it kind of makes sense while we're undecided on the best way. Maybe add a TODO?